index.js

```diff
@@ -76,33 +76,52 @@ async function createRoom() {
 async function createRoomAndStart() {
   const createAndStartButton = document.getElementById('create-and-start');
   const copyUrl = document.getElementById('copy-url');
+  const errorTitle = document.getElementById('error-title');
+  const errorDescription = document.getElementById('error-description');
 
   createAndStartButton.innerHTML = 'Loading...';
 
   room = await createRoom();
+  if (!room) {
+    errorTitle.innerHTML = 'Error creating room';
+    errorDescription.innerHTML =
+      "If you're developing locally, please doublecheck the README instructions.";
+    showError();
+  }
   copyUrl.value = room.url;
 
   showDemoCountdown();
 
-  callFrame.join({
-    url: room.url,
-    showLeaveButton: true,
-  });
+  try {
+    callFrame.join({
+      url: room.url,
+      showLeaveButton: true,
+    });
+  } catch (e) {
+    errorTitle.innerHTML = 'Error joining the call';
+    errorDescription.innerHTML = 'Please refresh the page and try again.';
+    showError();
+  }
 }
 
 async function joinCall() {
   const url = document.getElementById('url-input').value;
   const copyUrl = document.getElementById('copy-url');
   copyUrl.value = url;
 
-  await callFrame.join({
-    url: url,
-    showLeaveButton: true,
-  });
+  try {
+    await callFrame.join({
+      url: url,
+      showLeaveButton: true,
+    });
+  } catch (e) {
```

Comment:
Does it go into this block for you? I'm seeing this when I pass a random string as the Daily Url when I use the preview link. 
<img width="1494" alt="CleanShot 2021-05-19 at 18 30 51@2x" src="https://user-images.githubusercontent.com/18265845/118893828-1b434080-b8d1-11eb-8c6b-ec464945f9ca.png">

Reply:
It _did_! And this is why I should screen record everything, because now I feel like I'm losing my mind. I'll debug!

Reply:
https://www.loom.com/share/397a7f7e54f14d2d8d153c5903ef5df7

Reply:
https://www.loom.com/share/e9e2ccb53341403c982371689afb3458

Reply:
Thanks for the looms, super helpful. This can be another ticket (non-blocking) but it would be great to add a `disabled` attribute to the button instead of just styling it disabled. I actually didn't know it was disabled until seeing you describe it here. I thought it was just styled like a secondary button and didn't do anything. 😂 So that was part of the confusion. 

LGTM-- approving! Nice job.

Reply:
Thank you! I'm working on adding that disabled attribute (seems very quick), and will merge after.